core/src/test/kotlin/fr/sncf/osrd/stdcm/FullSTDCMTests.kt

```diff
@@ -266,6 +266,41 @@ class FullSTDCMTests {
             .run() ?: return
     }
 
+    /**
+     * The zone after the stop isn't available during the stop itself (and for a little while
+     * after). There is a solution if we lengthen the stop, but we need to properly account for the
+     * 20s margin where the signal must be green before the stop departure.
+     */
+    @Test
+    fun testConflictAfterStop() {
+        val infra =
+            Helpers.fullInfraFromRJS(Helpers.getExampleInfra("overlapping_routes/infra.json"))
+        val start = convertRouteLocation(infra, "rt.det.a1.nf->det.b1.nf", Offset(0.meters))
+        val stop =
+            convertRouteLocation(
+                infra,
+                "rt.det.a1.nf->det.b1.nf",
+                Offset(6_000.meters) // Within sight distance of a signal
+            )
+        val end = convertRouteLocation(infra, "rt.det.a1.nf->det.b1.nf", Offset(10_000.meters))
+        val zoneAfterStop = "zone.[det.center.2:INCREASING, det.center.3:DECREASING]"
+        val requirements = listOf(SpacingRequirement(zoneAfterStop, 0.0, 7_000.0, true))
+        val res =
+            STDCMPathfindingBuilder()
+                .setInfra(infra)
+                .setStartTime(0.0)
+                .setStartLocations(start.blockLocations)
+                .addStep(STDCMStep(stop.blockLocations, 5_000.0, true))
+                .setEndLocations(end.blockLocations)
+                .setBlockAvailability(makeBlockAvailability(requirements))
+                .setMaxRunTime(Double.POSITIVE_INFINITY)
+                .setMaxDepartureDelay(0.0)
+                .run()!!
+        assertTrue(res.stopResults.first().duration >= 5_000.0) {
+            "if a solution can be found without lengthening the stop, the test itself is broken"
```

Comment:
From the "comment" I expected:
```suggestion
        assertTrue(res.stopResults.first().duration > 5_000.0) {
            "if a solution can be found without lengthening the stop, the test itself is broken"
```

Reply:
Good catch, I accidentally included a local change I should have reverted. Fixed in https://github.com/OpenRailAssociation/osrd/pull/11271/commits/8f2263f3fccd1fde472d3eed77eb90c0c351a928